lib/voog/dtk/filemanager.rb

```diff
@@ -34,8 +34,10 @@ def read_manifest
     end
 
     def write_manifest(manifest)
-      if manifest["layouts"].is_a?(Array)
-        manifest["layouts"].sort! { |a, b| a["file"] <=> b["file"]}
+      %w(assets layouts).each do |k|
+        if manifest[k].is_a?(Array)
+          manifest[k].sort! { |a, b| a["file"] <=> b["file"] }
```

Comment:
Võiks garanteerida, et õhku ei lenda. Kui `'file'` võti ühel võrreldavatest puudub, siis annab `<=>` tagasi `nil` ja `sort!` feilib. 

```suggestion
           manifest[k].sort! { |a, b| a["file"] <=> b["file"] || 0 }
```

Pisidetail: kui interpolatsiooni ei kasuta, siis üldiselt on Rubys _string literal_ puhul kombeks `'single quotes'`.